test/Microsoft.IdentityModel.Tests/IdentityUtilities.cs

```diff
@@ -93,6 +94,99 @@ public static JwtSecurityToken CreateJwtSecurityToken(string issuer, string audi
             return new JwtSecurityToken(header, payload, header.Base64UrlEncode(), payload.Base64UrlEncode(), "" );
         }
 
+        public static string CreateSamlSecurityToken(SecurityTokenDescriptor securityTokenDescriptor, SecurityTokenHandler tokenHandler)
```

Comment:
Rename to CreateEncodedSaml.
The name distinguishes whether a token or string is returned.

Reply:
Fixed